HTML/nested.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nested Classes Example</title>
+</head>
+<body>
+    <h1>Nested Classes Example</h1>
+    <div id="output"></div>
+
+    <script>
+        class OuterClass {
+            constructor(name) {
+                this.name = name;
+            }
+
+            greet() {
+                console.log(`Hello from ${this.name}`);
+            }
+
+            class InnerClass {
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/16#issuecomment-2217711495>#fc2788</a><span> - 07/09/2024, 01:23 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The 'InnerClass' is defined inside the 'OuterClass' but is not properly scoped. JavaScript does not support nested classes in the way shown. This will cause a runtime error.
</div>
<div id="fix">
<b>Fix:</b> Move the 'InnerClass' definition outside of the 'OuterClass' and add a reference to it within 'OuterClass'.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -21,11 +21,11 @@
              console.log('Hello from ${this.name}');
          }

 -        class InnerClass {
 -            constructor(innerName) {
 -                this.innerName = innerName;
 -            }
 -
 -            innerGreet() {
 -                console.log('Hello from inner class, ${this.innerName}');
 -            }
 -        }
 +    }
 +
 +    class InnerClass {
 +        constructor(innerName) {
 +            this.innerName = innerName;
 +        }
 +
 +        innerGreet() {
 +            console.log('Hello from inner class, ${this.innerName}');
 +        }
      }

      const outer = new OuterClass('Outer');
```
</div>
</details>
</div>
<br>


</div>